hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -432,9 +432,9 @@ private static String getTmpSerializerFile() {
     return TMP_DIR + UUID.randomUUID().toString() + ".ser";
   }
 
-  private <T> T deSerializeOperationResult(String inputP, FileSystem fs) throws Exception {
-    Path inputPath = new Path(inputP);
-    InputStream inputStream = fs.open(inputPath);
+  private <T> T deSerializeOperationResult(HoodieLocation inputLocation,
```

Comment:
idk if its too late - but I think its worthwhile renaming `HoodieLocation` to just `StorageLocation` or `FileLocation` or `StoragePath`. `HoodieLocation` is not very intuitive to read.

Reply:
I think the renaming makes sense.  I've addressed the renaming in #10672 .